core/variant_call.cpp

```diff
@@ -1390,45 +1390,41 @@ bool Variant::has_method(const StringName &p_method) const {
 }
 
 Vector<Variant::Type> Variant::get_method_argument_types(Variant::Type p_type, const StringName &p_method) {
+	ERR_FAIL_INDEX_V(p_type, Variant::VARIANT_MAX, Vector<Variant::Type>());
 	const _VariantCall::TypeFunc &tf = _VariantCall::type_funcs[p_type];
 
 	const Map<StringName, _VariantCall::FuncData>::Element *E = tf.functions.find(p_method);
-	if (!E) {
-		return Vector<Variant::Type>();
-	}
+	ERR_FAIL_COND_V(!E, Vector<Variant::Type>());
```

Comment:
We discussed this in a review meeting, the `ERR_FAIL_INDEX_V` are good, but we're not confident that the change from `if (!E)` to an error condition is good. There's a risk that these methods might actually be used with non-existing methods at times and the code expects it to return an empty/default value.

Since it's not part of the fix for the reported bug, it's probably best to leave this part alone (same in other methods, only validate `p_type`).

Reply:
Removed changes.